src/Microsoft.AspNetCore.JsonPatch/Internal/PathHelpers.cs

```diff
@@ -13,7 +13,7 @@ internal static string NormalizePath(string path)
             // absolutely necessary, but it allows us to already catch mistakes
             // on creation of the patch document rather than on execute.
 
-            if (path.Contains("//") || path.Contains(" ") || path.Contains("\\"))
+            if (path.Contains("//"))
```

Comment:
Since you're here, can you change both the `.Contains()` and the `.StartsWith` below to either:
1. Use the `char`-based check
2. If not possible, pass in the explicit `StringComparison.Ordinal`

Reply:
Might be worthwhile to change the name to `ValidateAndNormalizePath`. Also the `StartsWith` below needs a `StringComparison.Ordinal`

Reply:
Updated `StartsWith`. Going to leave `Contains` as is because I think it is cleaner than doing `IndexOf` with StringComparison.Ordinal.

Reply:
Ah, didn't realize that `.Contains()` always does ordinal case-sensitive, so this one is ok.

Reply:
`Contains` does the right thing - https://referencesource.microsoft.com/#mscorlib/system/string.cs,2129